test/sizes.gzip.json

```diff
@@ -0,0 +1,43 @@
+{
+  "amp.js": "92.33 kB",
```

Comment:
sorry, meant to keep the .txt file around and move forward with the .json.

will make the changes to keep the .txt.